catalogue/bpf/tests/LB+fcas-ctrlcvg-once+once-scas.litmus

```diff
@@ -0,0 +1,33 @@
+BPF LB+fcas-ctrl-once+once-scas
```

Comment:
Having identical names for different tests will confuse some tools
```suggestion
BPF LB+fcas-ctrlcvg-once+once-scas
```

Reply:
Thanks for catching this,
fixed in next version